src/pages/certification/components/AuditorRunTestForm/AuditorRunTestForm.tsx

```diff
@@ -293,7 +303,7 @@ const AuditorRunTestForm: React.FC<IAuditorRunTestForm> = ({
 
   return (
     <div>
-      <Form form={form} onSubmit={formHandler}>
+      <form onSubmit={form.handleSubmit(formHandler)}>
```

Comment:
Just to be aware - Is there any particular reason why this update is in place? Is there any impact on the component.

Reply:
This component was only used here and it doesn't have much behavior, so I didn't see the need to keep it